DEVELOPING.md

```diff
@@ -0,0 +1,179 @@
+# Developing
+
+## Getting started
```

Comment:
One comment since `Levels of Testing` is capitalized should this be `Getting Started`?

Reply:
I'll update in the other direction 👍